src/main/java/com/google/sps/data/Tag.java

```diff
@@ -14,4 +15,15 @@ public Tag() {
   public Tag(String id) {
     super(id);
   }
+  
+  public Tag(String id, String name, boolean hidden, String type) {
+    super(id);
+    this.name = name;
+    this.hidden = hidden;
+    this.type = type;
+  }
+
+  public String toString() {
+    return "Tag[id=" + super.id + ", name=" + name + ", hidden=" + String.valueOf(hidden) + ", type=" + type + "}";
```

Comment:
what is this being used for? Also it is starting with [ and ending with }.

Reply:
I thought it might be useful for debugging (`toString` is used in all the printing functions)